cmd/cloudPrivateParentDetails.go

```diff
@@ -18,9 +18,8 @@ package cmd
 import (
 	"fmt"
 
+	apiTypes "github.com/liquidweb/liquidweb-cli/types/api"
 	"github.com/spf13/cobra"
-
-	"github.com/liquidweb/liquidweb-cli/types/api"
```

Comment:
similar to my last comment about this. I won't make any other mentions of it. If its something `vim-go` or similar is doing automatically I suppose don't worry about it.

Reply:
gah, vim-go is doing that to me... I'll run some tests on formatting and see if I can make it stop :(